components/layout_2020/flow/mod.rs

```diff
@@ -667,9 +666,9 @@ fn layout_in_flow_non_replaced_block_level_same_formatting_context(
             // the margin should have been included in the parent (or some ancestor).
             // The lookahead should stop for actual clearance, not just for `clear`.
             let collapsible_with_parent_start_margin = collapsible_with_parent_start_margin.expect(
-                "We should know whether we are collapsing the block start margin with the parent \
-                when laying out sequentially",
-            ).0 && style.get_box().clear == Clear::None;
+                 "We should know whether we are collapsing the block start margin with the parent \
+                 when laying out sequentially",
+             ).0 && style.get_box().clear == Clear::None;
```

Comment:
The white space only changes here indicate that you haven't run './mach fmt' locally before submitting this change. Please do that.

Reply:
I ran ./mach fit multiple times, still this piece of code remain neglected. I've fixed the it manually.

Reply:
It's pretty unlikely that `./mach fmt` should fail. What is the output you see when running `./mach fmt`? Perhaps there is a problem with the installation of your Rust toolchain?

Reply:
After running ./mach fmt, while most of the codebase's formatting issues are addressed, and indentation changes are correctly applied, there are still some sections of the code that remain unformatted. The command completes without any output messages.

Reply:
What do you mean by "remain unformatted." The changes that `./mach fmt` does should result in the proper formatting of all Rust code in Servo. Can you please paste the output of running `./mach fmt` and  and then `./mach test-tidy` (which tests that `./mach fmt` was done)? If `./mach test-tidy` fails after running `./mach fmt` then there is definitely a bug that should be fixed. It's quite likely if you modify the formatting of Rust code after running `./mach fmt` though, `./mach test-tidy` will (and should!) fail.

Reply:
<img width="500" alt="Screenshot 2024-03-26 at 00 57 53" src="https://github.com/servo/servo/assets/137225906/1ac1090a-64e4-48ab-9908-cf00d6b6431b">

Reply:
Even while I raised this PR and pushed changes, test-tidy reported no errors.

Reply:
Don't change this. Regardless of whether `./mach fmt` addresses this or not.